Essentials/src/main/java/com/earth2me/essentials/commands/Commanddelhome.java

```diff
@@ -37,18 +39,24 @@ public void run(final Server server, final CommandSource sender, final String co
 
         if (expandedArg.length > 1 && (user == null || user.isAuthorized("essentials.delhome.others"))) {
             user = getPlayer(server, expandedArg, 0, true, true);
-            name = expandedArg[1];
+            name = expandedArg[1].toLowerCase(Locale.ENGLISH);
         } else if (user == null) {
             throw new NotEnoughArgumentsException();
         } else {
-            name = expandedArg[0];
+            name = expandedArg[0].toLowerCase(Locale.ENGLISH);
         }
 
-        if (name.equalsIgnoreCase("bed")) {
+        if (name.equals("bed")) {
             throw new Exception(tl("invalidHomeName"));
         }
 
-        user.delHome(name.toLowerCase(Locale.ENGLISH));
+        final HomeModifyEvent event = new HomeModifyEvent(sender.getUser(ess), user, name, user.getHome(name), false);
+        Bukkit.getServer().getPluginManager().callEvent(event);
+        if (event.isCancelled()) {
+            return;
```

Comment:
Similar to #5304, cancellation feedback would make it clearer to users why the home hasn't been modified.